daml/Marketplace/Custody/Model.daml

```diff
@@ -124,9 +123,11 @@ template LifecycleRequest
               spot label dt = pure $ fromSomeNote
                 ("Lifecycle: Underlying data '" <> label <> "' missing on " <> show dt)
                 (Map.lookup label fixings >>= Map.lookup dt)
-              election : Claim Observation Date Id -> Claim Observation Date Id -> Claim Observation Date Id
-              election _ _ = deserialize choice -- will throw if the claims don't match
-          Lifecycle.Result{remaining, pending} <- Lifecycle.lifecycle spot election (deserialize description.claims) t
+              election = (True, deserialize choice)
```

Comment:
With the updates to `contingent-claims`, the exercising/lifecycling have been split into separate functions.

I've merged this with the aim of minimising the delta, which will also keep the functionality and API the same, but we may want to re-visit our approach to lifecycling in future; currently it has to be initiated by the customer, but we could now. E.g. create a trigger run by the custodian that attempts to lifecycle stuff automatically. This makes sense from a practical point of view (in the real world you don't need to request coupon or dividend payments - they happen automatically).